client/components/Toggle/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import classNames from 'classnames'
+import './style.scss'
+
+export default function Toggle({ value, onChange, readOnly }) {
+    const onClick = () => onChange({ target: { value: !!value } })
```

Comment:
`!!value` will cast the value to a boolean. I guess you meant `!value` to inverse the current value.